.circleci/config.yml

```diff
@@ -1177,33 +1286,19 @@ workflows:
   generate-snapshot:
     when: << pipeline.parameters.generate_snapshots >>
     jobs:
-      - run-test-ios-17:
-          xcode_version: "15.2"
-      - run-test-ios-16:
-          xcode_version: '14.3.0'
-          install_swiftlint: false
-      - run-test-ios-15:
-          xcode_version: '14.3.0'
-          install_swiftlint: false
-      - run-test-ios-14:
-          xcode_version: '14.2.0'
-          install_swiftlint: false
-      - run-test-ios-13:
-          xcode_version: '14.2.0'
-          install_swiftlint: false
+      - run-test-ios-17
+      - run-test-ios-16
+      - run-test-ios-15
+      - run-test-ios-14
+      - run-test-ios-13
```

Comment:
this is the result of moving the xcode versions up, the workflows are now very clean